README.md

```diff
@@ -1,9 +1,13 @@
 # Poetry Website
 
-This is the source of the official [Poetry website](https://python-poetry.org).
+[![Poetry](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/python-poetry/website/main/static/badge/v0.json)][official website]
```

Comment:
Hmm, I'm not sure I have the experience to make that work. The way it's implemented now is how [Ruff's badge works](https://github.com/charliermarsh/ruff/tree/main/assets/badge), and it seemed like a good approach?

Reply:
@Secrus It looks like this could in fact work (from the preview website, I validated that `/badge/v0.json` is accessible):

- https://website-9cvbcandi-python-poetry.vercel.app/badge/v0.json

So the link would look like:

```md
[![Poetry](https://img.shields.io/endpoint?url=https://python-poetry.org/badge/v0.json)][official website]
```

Is this what you prefer? I _personally_ think routing it to GitHub is pretty simple, but if you'd prefer we point shields.io at the Poetry website instead, I'm happy to make the changes to the PRs.

Reply:
@Secrus 4e9bac3 updates the badge URL to point to python-poetry.org hosted badge JSON.

Let me know if there is anything else you'd like updated.

Reply:
@Secrus Just checking it to see if you wanted any more changes here.